packages/swarm-components/src/Select.jsx

```diff
@@ -20,35 +20,44 @@ const Select = (props: Props) => {
         id,
 		label,
         name,
-		required,
+		requiredText,
 		...other
 	} = props;
 
+	const selectState = disabled
+		? 'disabled'
+		: (error ? 'error' : 'default');
```

Comment:
I also don't think prettier is running on commit. Might want to look into that separately.